dpctl/tensor/_elementwise_common.py

```diff
@@ -927,3 +928,142 @@ def __call__(self, o1, o2, /, *, out=None, order="K"):
         )
         _manager.add_event_pair(ht_, bf_ev)
         return out
+
+    def _inplace_op(self, o1, o2):
+        if self.binary_inplace_fn_ is not None:
```

Comment:
Probably easier to read, and less indentation if we check and raise if "self.binary_in_place_fn is None` and then proceed not indented.

Reply:
Good idea, I've made the change.